src/components/document/document-workspace.tsx

```diff
@@ -188,6 +190,23 @@ export class DocumentWorkspaceComponent extends BaseComponent<IProps, {}> {
     }
   }
 
+  private handleCopyDocument = (document: DocumentModelType) => {
+    this.stores.ui.prompt(`Give your workspace copy a new name:`,
+                          `Copy of ${document.title || this.stores.problem.title}`, `Copy Workspace`)
```

Comment:
```suggestion
                          `Copy of ${document.title || this.stores.problem.title}`, `Copy Document`)
```

Reply:
Likewise, `Workspace` came from the design specs for the new modals/pop-ups. Should this be clarified also again?

Reply:
Same answer as above. If it's from the spec, go with the spec. I didn't realize there was a spec available for this.

Reply:
Sounds good to me for both!